packages/manager/modules/pci/src/projects/project/data-processing/notebooks/notebook-details/notebook-details.html

```diff
@@ -0,0 +1,174 @@
+<oui-header
+    class="notebook-title__header"
+    data-heading="{{ $ctrl.notebook.spec.name }}"
+    data-description="{{ $ctrl.notebook.id }}"
+>
+    <oui-guide-menu
+        data-text="{{ :: 'pci_project_guides_header' | translate }}"
+    >
+        <oui-guide-menu-item
+            data-ng-repeat="guide in $ctrl.guideUrl.data_processing track by $index"
+            data-href="{{:: guide.url }}"
+            data-on-click="$ctrl.trackClick($ctrl.guideTrackingSectionTags.data_processing + guide.tracking)"
+            data-external
+        >
+            <span
+                data-translate="{{:: 'pci_project_guides_header_' + guide.key }}"
+            ></span>
+        </oui-guide-menu-item>
+    </oui-guide-menu>
+
+    <oui-header-tabs>
+        <oui-header-tabs-item
+            data-state="pci.projects.project.data-processing.notebooks.details"
+            data-state-params="{ notebookId: $ctrl.notebook.id }"
+            data-active
+            ><span
+                data-translate="data_processing_notebooks_details_dashboard_label"
+            ></span>
+        </oui-header-tabs-item>
+    </oui-header-tabs>
+</oui-header>
+<div class="notebook-details__content">
```

Comment:
**nitpick:** Bootstrap classes `col-sm`, `col-md`,`col-lg` and `col` can only exist as a directly child of an element with `.row`.  So please have a row and then size the columns using the helpers.